release/README.md

```diff
@@ -58,21 +63,68 @@ tweak as needed.
 - A change that affects all libraries should only be documented in the
   `Common Libraries` section.
 
-### Run checkers
+#### Run checkers
 
 ```bash
 ci/cloudbuild/build.sh -t checkers-pr
 ```
 
-### Send a PR with all these changes
+#### Send a PR with all these changes
+
+```bash
+git add .
+git checkout -b release-changelog
+git commit -am "docs(release): update changelog for XX-20XX release" // Change this to the month of the release
+git push
+```
+
+## 2. Bump the version number in `main`
+
+Working in your fork of `google-cloud-cpp`: bump the version numbers to the
+*next* version, i.e., one version past the release you just did above. Then send
+the PR for review against `main`. You need to:
+
+### a. Make the following changes
+
+- In the top-level `CMakeLists.txt` file:
+  - a. Increment the version number in the `project()` function.
+  - b. Set the pre-release version (PROJECT_VERSION_PRE_RELEASE) to "rc".
+- In the `CHANGELOG.md` file:
+  - Add a "vX.Y.Z - TBD" header, corresponding to the new version number.
+
+### b. Run the following script
+
+- Update the ABI baseline to include the new version numbers in the inline
+  namespace by running `ci/cloudbuild/build.sh -t check-api-pr`. This will leave
+  the updated ABI files in `ci/abi-dumps`, and also update the
+  `google/cloud/internal/version_info.h` file. This will step will take a while.
+
+### c. Send the PR for review
 
-In general, do not create the release branch before this PR is *merged*. We want
-to create the release from a stable point in the default branch (`main`), and we
-want this point to include the updated release notes and API baselines. There
-may be exceptions to this guideline, you are encouraged to use your own
-judgment.
+**NOTE:** Do NOT submit this before PR before the PR in step 1 is *merged*.
 
-## Creating the release
+```bash
+git add .
+git checkout -b bump-rc
+git commit -am "chore: version bump to X.XY.Zrc" // Change this to the the correct version
+git push
+```
+
+**NOTE:** The Renovate bot will automatically update the Bazel deps in the
+quickstart `WORKSPACE.bazel` files after it sees the new release published.
+Watch for this PR to come through, kick off the builds, approve, and merge it.
+
+## 3. Creating the release
+
+**NOTE:** Do NOT create the release branch before the PR created in step 1 and 2
```

Comment:
Only step 1 is blocking.  I think the ideal order is something like:

- Create the PR for CHANGELOG
- While that goes through review and builds: start the build for the version bump
- Merge the PR for the CHANGELOG
- Create the release
- Send the PR to bump the version number

Reply:
Done